tests/unit/test.linter.js

```diff
@@ -577,6 +577,7 @@ describe('Linter.textOutput()', () => {
   });
 
   it(
+    // eslint-disable-next-line jest/valid-title
     oneLine`should remove columns, description, and lines when terminal is < 60 columns wide`,
```

Comment:
@wagnerand I'm wondering why there is a `oneLine` here, my guess is that this used to be a multiline string before we introduced prettier, then prettier "decided that it liked it more as a single line" :-P and we never noticed it until now (but that is just a guess, I didn't do the actual "git archeology" to double-check that).

If we remove `oneLine` and turn it into a non-template string, is that enough to avoid disabling `eslint-disable-next-line jest/valid-title` for this line?

Reply:
I thought we had that because otherwise a different check would fail because the line is longer than 80 characters.

Reply:
5cd597c0